blackbox/docs/administration/hba.txt

```diff
@@ -23,12 +23,8 @@ See: :ref:`applying-cluster-settings`.
 Allowing Authentication to CrateDB
 ==================================
 
-.. NOTE::
-
-   This feature only concerns connections over :ref:`postgres_wire_protocol`.
-
-Client access and authentication is configured by the ``auth.host_based.config``
-setting in the ``crate.yml`` file.
+Client access and authentication over :ref:`protocols` is configured via the
```

Comment:
Client access and authentication over HTTP and Postgres Protocol (see :doc:protocols/index) ...